sections/Titans stronghold.tex

```diff
@@ -0,0 +1,174 @@
+  \newcommand{\silver}[0]{\svg[12]{silver}}
+  \newcommand{\golden}[0]{\svg[12]{golden}}
+  \newcommand{\azure}[0]{\svg[12]{azure}}
+
+  \begin{tabularx}{\linewidth}{p{0.15\linewidth}XXXX} & \darkcell{Easy} & \darkcell{Normal} & \darkcell{Hard} & \darkcell{Impossible}\\
+    \darkcell[1.4]{1 player} 
+        & \lightcell[1.4]{3\bronze 2\silver 1\golden 1\azure}
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 1\azure}
+        & \lightcell[1.4]{2\bronze 2\silver 2\golden 2\azure}
+        & \lightcell[1.4]{1\bronze 2\silver 2\golden 3\azure}\\
+    \darkcell[1.4]{2 players} 
+	    & \lightcell[1.4]{5\bronze 5\silver 3\golden 1\azure}
+	    & \lightcell[1.4]{4\bronze 5\silver 3\golden 2\azure}
+	    & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure}
+	    & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure}\\
+    \darkcell[1.4]{3 players} 
+        & \lightcell[1.4]{8\bronze 7\silver 4\golden 2\azure}
+        & \lightcell[1.4]{6\bronze 7\silver 5\golden 3\azure}
+        & \lightcell[1.4]{2\bronze 3\silver 4\golden 3\azure \linebreak
+        	Then\linebreak 
+        	2\bronze 4\silver 3\golden 2\azure}
+        & \lightcell[1.4]{1\bronze 3\silver 5\golden 3\azure \linebreak
+        	Then\linebreak 
+        	1\bronze 4\silver 5\golden 3\azure}\\
+    \darkcell[1.4]{4 players} 
+        & \lightcell[1.4]{10\bronze 10\silver 6\golden 2\azure}
+        & \lightcell[1.4]{8\bronze 10\silver 6\golden 4\azure}
+        & \lightcell[1.4]{2\bronze 5\silver 5\golden 3\azure \linebreak
+        	Then\linebreak 
+        	2\bronze 5\silver 5\golden 3\azure}
+        & \lightcell[1.4]{1\bronze 5\silver 7\golden 4\azure \linebreak
+        	Then\linebreak 
+        	1\bronze 5\silver 7\golden 4\azure}\\
+	\darkcell[1.4]{5 players} 
+	  	& \lightcell[1.4]{6\bronze 6\silver 4\golden 1\azure \linebreak
+	  		Then\linebreak 
+	  		7\bronze 6\silver 3\golden 2\azure}
+	  	& \lightcell[1.4]{5\bronze 6\silver 4\golden 2\azure \linebreak
+	  		Then\linebreak 
+	  		5\bronze 6\silver 4\golden 3\azure}
+	  	& \lightcell[1.4]{3\bronze 6\silver 6\golden 4\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 6\silver 6\golden 4\azure}
+	  	& \lightcell[1.4]{1\bronze 6\silver 8\golden 6\azure \linebreak
+	  		Then\linebreak 
+	  		2\bronze 6\silver 8\golden 5\azure}\\
+	\darkcell[1.4]{6 players} 
+	  	& \lightcell[1.4]{8\bronze 8\silver 4\golden 1\azure \linebreak
+	  		Then\linebreak 
+	  		7\bronze 7\silver 5\golden 2\azure}
+	  	& \lightcell[1.4]{6\bronze 8\silver 4\golden 3\azure \linebreak
+	  		Then\linebreak 
+	  		6\bronze 7\silver 5\golden 3\azure}
+	  	& \lightcell[1.4]{3\bronze 8\silver 7\golden 5\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 7\silver 8\golden 4\azure}
+	  	& \lightcell[1.4]{3\bronze 7\silver 11\golden 6\azure \linebreak
+	  		Then\linebreak 
+	  		3\bronze 8\silver 10\golden 6\azure}\\
+  \end{tabularx}
+}
+
+\vspace{3em}
+
+\center\includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
```

Comment:
Please describe the images. Right now, they don't explain the number of players. There should be something like "X Player Tile Setup".

Reply:
```
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.2\paperwidth]{\_assets/maps/titans-1.png}
  \captionof{figure}{1 player}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-2.png}
  \captionof{figure}{2 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-3.png}
  \captionof{figure}{3 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-4.png}
  \captionof{figure}{4 players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-5.png}
  \captionof{figure}{5 Players}
\end{minipage}
\begin{minipage}{0.4\paperwidth}
  \centering
  \includegraphics[width=0.4\paperwidth]{\_assets/maps/titans-6.png}
  \captionof{figure}{6 players}
\end{minipage}
```

<img width="595" alt="Screenshot 2024-06-25 at 9 22 51" src="https://github.com/qwrtln/Homm3BG-mission-book/assets/1928291/237c13d8-ff27-4eb4-9f83-49861bb3dd59">